### PR TITLE
Feature/improve bulk order

### PR DESCRIPTION
**Pull request by karacurt**
- Set maximum amount to 10 
- Now set amount through a slider
- New images for boosters
- Amount images to be show together with boosters

image

### CODE REVIEW

**Comment by aviggiano** [comment]
The "slider amount" should have a number indicating how many items are there

**Comment by karacurt** [comment]
> The "slider amount" should have a number indicating how many items are there

there is a tool tip, sorry i didn't print it

image

**Comment by aviggiano** [comment]
> there is a tool tip, sorry i didn't print it

I think it's not clear enough

**Comment by karacurt** [comment]
> > there is a tool tip, sorry i didn't print it
> 
> I think it's not clear enough

how about know?

image

File: components/common/Marketplace/FormCreateOrder.tsx [hunk 1]
```diff
@@ -200,21 +201,20 @@ const FormCreateOrder: React.FC<Props> = ({ setOrder, isBulkOrder, setIsBulkOrde
...
-                    const { value } = obj.target
-                    if (value.length > 15) return
-                    if (Number(value) < 2) return
-                    setAmount(Math.floor(Math.abs(Number(value))))
-                  }}
-                />
+                <AmountFilter>
+                  <input
+                    onChange={(obj) => {
+                      const { value } = obj.target
```
Comment (pedrohw12) [inline components/common/Marketplace/FormCreateOrder.tsx]: I think that could be better to create a function and call it there in the 'onChange' prop. What do you think ?

**Comment by aviggiano** [comment]
> how about know?

It's a bit weird the positioning. Maybe something like this? https://api.flutter.dev/flutter/material/Slider-class.html

File: components/common/Marketplace/FormCreateOrder.tsx (hunk 1 above)
Comment (karacurt) [reply components/common/Marketplace/FormCreateOrder.tsx]: sounds great

**Comment by karacurt** [comment]
image

**Comment by aviggiano** [comment]
OMG SO BEAUTIFUL !!!!!
image

**Pull request merged**